include/simdjson/parsedjson_iterator.h

```diff
@@ -15,12 +15,47 @@
 #include "simdjson/internal/jsonformatutils.h"
 
 namespace simdjson {
+// We moved the definitions of the ctor inline for this deprecated
+// class to avoid spurious warnings under Visual Studio.
+
 
 class [[deprecated("Use the new DOM navigation API instead (see doc/usage.md)")]] dom::parser::Iterator {
 public:
-  inline Iterator(const dom::parser &parser) noexcept(false);
```

Comment:
Clever way to fix, and seems totally fine to me!

Reply:
I feel better about this way of doing things that the removal of deprecated under Visual Studio. I feel like Visual Studio should be a first citizen, as much as possible, with top-notch support...

Reply:
The price we pay is that we make the code a bit uglier, but given that this is deprecated code...

Reply:
I tried fixing it by moving the [[deprecated]] to the definition (which is in dom::parser). But I agree, it's deprecated code and I don't want to futz with it.